app/models/main.py

```diff
@@ -1687,6 +1714,16 @@ def serialize(self):
 
         return purge_nulls_from_data(data)
 
+    def _parent_brief_fields(self, parent_brief):
+        return {
+            **self._unpack_fields(self.PARENT_BRIEF_FIELDS, parent_brief),
+            **{'framework': self._unpack_fields(self.FRAMEWORK_FIELDS, parent_brief)}
+        }
+
+    @staticmethod
+    def _unpack_fields(required_fields, collection):
```

Comment:
_Personally_ probably wouldn't bother splitting this out and just write it inline instead, even if it's repeated. Means reader doesn't have to go & find the implementation of the function to see what it's doing, and doesn't pollute any namespaces...